sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -3405,9 +3453,33 @@ public partial class EncryptionSettingsGroup
         public static bool operator !=(Azure.ResourceManager.Compute.Models.ExecutionState left, Azure.ResourceManager.Compute.Models.ExecutionState right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class GalleryApplicationCustomAction
+    {
+        public GalleryApplicationCustomAction(string name, string script) { }
+        public string Description { get { throw null; } set { } }
+        public string Name { get { throw null; } set { } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.Compute.Models.GalleryApplicationCustomActionParameter> Parameters { get { throw null; } }
+        public string Script { get { throw null; } set { } }
+    }
+    public partial class GalleryApplicationCustomActionParameter
```

Comment:
I think we'd better not have this model name end with `Parameter`.

Reply:
This ends with `Parameter` because it stands for a configurable "parameter" - not like the case of request body's parameter.
For instance, there is another model with a property `Parameters`, and the type of the property is a list of this "GalleryApplicationCustomActionParameter". Therefore I believe it is reasonable that this ends with Parameter